static/src/css/vendor/bootstrap/overrides/_forms.scss

```diff
@@ -27,23 +27,23 @@
 .dropdown-dark,
 .dropdown-menu-dark {
   .form-control {
-    background-color: darken($color__black--600, 10);
-    color: $color__white;
-    border: 1px solid darken($color__black--600, 10);
+    background-color: darken($color__carbon--60, 10);
+    color: $color__spacesuit-white;
+    border: 1px solid darken($color__carbon--60, 10);
```

Comment:
I am not sure if we should technically be using darken/lighten on the hds colors